### DISCUSSION TITLE
🚫 Error loading: This IP address has been blocked by YouTube (429).

### DISCUSSION THREAD

**Discussion by lemonchan1992** [jagrosh/MusicBot#870] [Troubleshooting] [answered]
Bot was worked well just now until i try again 10mins ago , it suddenly appear this error code, it means the bot ip (heroku) by blocked from youtube?

**Comment by MichailiK** [comment, marked as answer]
Yes. This is one of several reasons we advice in our [24/7 Hosting guide](https://jmusicbot.com/hosting) **not to use Heroku**.
